README.md

```diff
@@ -1,39 +1,40 @@
 [![Travis CI](https://travis-ci.org/conversationai/perspectiveapi-authorship-demo.svg?branch=master)](https://travis-ci.org/conversationai/perspectiveapi-authorship-demo) [![npm version](https://badge.fury.io/js/%40conversationai%2Fperspectiveapi-authorship-demo.svg)](https://badge.fury.io/js/%40conversationai%2Fperspectiveapi-authorship-demo)
 
-# An authorship experience demo of the perspective API
+# An authorship experience demo for Perspective API
 
-This repository contains example code for an authorship experience that provides feedback as you type using the [Perspective API](http://www.perspectiveapi.com/). It is a web app written in
-[Angular2](https://angular.io/)/[TypeScript](https://www.typescriptlang.org/) and uses a [node express server](https://expressjs.com/). This project shows how to have a development and a production
-build environment with deployment tools based on [Docker](https://www.docker.com/). This makes it easy be deployed to cloud services such as Google Cloud's flexible appengine environment, as well as other cloud computing providers.
+This repository contains example code for an authorship experience that provides feedback as you type using [Perspective API](http://www.perspectiveapi.com/). It is a web app written in
+[Angular2](https://angular.io/)/[TypeScript](https://www.typescriptlang.org/) and uses a [Node Express server](https://expressjs.com/). This project shows how to have a development and a production
+build environment with deployment tools based on [Docker](https://www.docker.com/). You can then deploy to cloud services such as [Google Cloud's App Engine](https://cloud.google.com/appengine/),
+as well as other cloud computing providers.
 
-The app uses the [Perspective API](http://www.perspectiveapi.com/) to score text, via a
-the [perspectiveapi-simple-server](https://github.com/conversationai/perspectiveapi-simple-server).
+The app uses the [Perspective API](http://www.perspectiveapi.com/) to score text, via
+[Simple Server for Perspective API](https://github.com/conversationai/perspectiveapi-simple-server).
```

Comment:
I think it's more understandable to call it by the name of the repo (perspectiveapi-simple-server).

Reply:
I've changed it back

Reply:
Thanks! Could you change it back in the other places too?

Reply:
I don't see it written anywhere else, that should have been the only change.

Reply:
I see it on line 27, 37, and 54. Apologies if you already edited this, github's UI for viewing changes can be confusing at times!

Reply:
It really can be!! I think it should really, actually be fixed this time.

Reply:
Thank you!